### DISCUSSION TITLE
Monkey watching politely

### DISCUSSION THREAD

**Discussion by kaitian996** [vuejs/pinia#1550] [Help and Questions]

**Comment by kaitian996** [comment]


**Reply by timi137137** [reply]
![image](https://user-images.githubusercontent.com/36812439/184869377-3055ca5a-f471-4540-aa55-d1d004c085b6.png)
xswlyuanl原来你还问了隔壁核心的人，跟我说的不是一模一样么，
![image](https://user-images.githubusercontent.com/36812439/184869467-2aae4357-cfcd-4458-bc3f-308d45205e7c.png)
官方都跟你说了可以，自己学艺不精说人解决不了问题，还说我态度不好
别出来祸害人了求求你了

**Reply by kaitian996** [reply]
你看看这两个问题是一个问题吗？core里面问的是如何让reactive对象属性依赖一个computed，pinia里面问的是为什么store不改变computed会重新调用，求求你了断断句吧，

**Reply by zuzuzujjj** [reply]
@timi137137 有不懂来提问很正常，怎么能说别人学艺不精结解决不了问题，出来祸害人呢，会就耐心讲解，不会就不会，说你态度不好就反思一下自己有没有这回事，有责改之，无则加勉即可。

**Reply by timi137137** [reply]
> @timi137137 有不懂来提问很正常，怎么能说别人学艺不精结解决不了问题，出来祸害人呢，会就耐心讲解，不会就不会，说你态度不好就反思一下自己有没有这回事，有责改之，无则加勉即可。

你说的并没错，但是我也没做错，我只是对他代码有疑惑，并告诉他解决方法，他说我态度不好。我只能说有人回答你问题你还应该感到庆幸，我也没骂人也没说你一定是错，我说的是“应该是这里的问题”我也没说一定对是吧，怎么就态度不好了，可笑可笑

**Reply by timi137137** [reply]
做人，你敬我一尺我敬你一丈，你不尊重我我干什么尊重你

**Reply by timi137137** [reply]
@suanzi-yi 
![image](https://user-images.githubusercontent.com/36812439/184876805-e0bdc51d-f0f7-4a81-a6e8-ccd8d40ddc05.png)
![image](https://user-images.githubusercontent.com/36812439/184876837-de256511-fc57-4f37-a0af-171bbac4fef8.png)
![image](https://user-images.githubusercontent.com/36812439/184876870-07cd2342-039a-46dc-a363-b5cd7d3d78b1.png)
回家多读书吧哈，别出来祸害人，出来之前好好学学怎么待人处事

**Reply by kaitian996** [reply]
哈哈，你别删评论，你那叫尊重人？张口就是 
你的代码真奇怪
![79d3a9dc4231a16dc98cb1581c5e12d](https://user-images.githubusercontent.com/85500608/184876767-b5080ca1-1c9f-4695-86b0-516c175ebaab.jpg)
我确实没学会，学会了就不问了，但是你跟人说话是这么说的？那我只能说：如果你这么想那就代表你没学会尊重人

**Reply by timi137137** [reply]
> 哈哈，你别删评论，你那叫尊重人？张口就是 你的代码真奇怪 ![79d3a9dc4231a16dc98cb1581c5e12d](https://user-images.githubusercontent.com/85500608/184876767-b5080ca1-1c9f-4695-86b0-516c175ebaab.jpg) 我确实没学会，学会了就不问了，但是你跟人说话是这么说的？那我只能说：如果你这么想那就代表你没学会尊重人

是呢，你确实没学会呢，你看看我的再看看你的代码
let const都没分清楚
vuex基础你肯定也没学过吧
可笑可笑

**Reply by timi137137** [reply]
你学会了吗？没有吧？得了不跟你这种人斗争了，反正问题解决了，记得回去好好学习，别丢你公司脸

**Reply by kaitian996** [reply]
我不跟你争，我才大二，时间还有很多，慢慢学就是了，你无非多几年工作经验，当我有能力能给别人解决问题时，我绝不会像你这样高高在上，丰硕的麦子总是谦卑的低下头颅。

**Reply by zuzuzujjj** [reply]
@timi137137 能力不是炫耀的资本，你仗着有经验确实有些咄咄逼人，希望自己能意识到这一点吧。 @suanzi-yi 看好你，学无止境，现在不会的慢慢学习就可以了，有问题大胆提出，总有人会耐心解答问题的，因为他们才是有经验的前辈。

**Reply by timi137137** [reply]
> 我不跟你争，我才大二，时间还有很多，慢慢学就是了，你无非多几年工作经验，当我有能力能给别人解决问题时，我绝不会像你这样高高在上，丰硕的麦子总是谦卑的低下头颅。

可惜我才初一
